include/mbgl/style/conversion/function.hpp

```diff
@@ -200,7 +200,7 @@ struct Converter<CameraFunction<T>> {
 
 template <class T, class V>
 Result<optional<T>> convertDefaultValue(const V& value) {
-    auto defaultValueValue = objectMember(value, "defaultValue");
+    auto defaultValueValue = objectMember(value, "default");
```

Comment:
Can you investigate why this mistake didn't trigger test failures? Contrary to my earlier statement, we do have tests which I would have expected to fail due to this, added in https://github.com/mapbox/mapbox-gl-js/commit/e328b807d9ad39a8194e9c13ab28ee1b16a8ce56.